src/plugins/index_pattern_field_editor/public/components/preview/field_preview.tsx

```diff
@@ -71,7 +71,13 @@ export const FieldPreview = () => {
   }, [reset]);
 
   return (
-    <div className="indexPatternFieldEditor__previewPannel">
+    <div
+      className="indexPatternFieldEditor__previewPannel"
+      tabIndex={0}
```

Comment:
This `tabIndex` is for the scrollable area of the flyout panel. cc @myasonik 

<img width="447" alt="Screenshot 2021-08-09 at 09 38 54" src="https://user-images.githubusercontent.com/2854616/128673387-3c2e43ac-8793-45cc-9638-c1d9ded22497.png">